### PR TITLE
Document myos.sh (Comments)

### PR DESCRIPTION
**Pull request by dimensi0n**
As promised, this is my pull request to make the myos script easier to understand.

### CODE REVIEW

File: myos.sh [hunk 1]
```diff
@@ -1,92 +1,117 @@
...
 SUDO=$(if docker info 2>&1 | grep "permission denied" >/dev/null; then echo "sudo -E"; fi)
 
+# Commands descriptions
 initHelp="init <envName> [options]"
 createHelp="create <envName> [options]"
 connectHelp="connect <envName> [options]"
 removeHelp="remove <envName> [options]"
 restartHelp="restart <envName> [options]"
 
+# What to do if there is an errer
```
Comment (rylandg) [inline myos.sh]: Small typo errer->error

File: myos.sh [hunk 2]
```diff
@@ -1,92 +1,117 @@
...
-  ssh-add $MYOS_DIR/myos-key
+    if [ ! -f $MYOS_DIR/authorized_keys ]; then
+        mkdir -p $MYOS_DIR
+        ssh-keygen -f $MYOS_DIR/myos-key -t rsa -N ''
+        cp $MYOS_DIR/myos-key.pub $MYOS_DIR/authorized_keys
+    fi
+    ssh-add $MYOS_DIR/myos-key
 }
 
+# What to do if the command is unknown
```
Comment (rylandg) [inline myos.sh]: I think this comment isn’t needed.

File: myos.sh [hunk 3]
```diff
@@ -1,92 +1,117 @@
...
 }
 
 enforceArgs $# 1 "MyOS"
 
+# Set command :
+# - Example : "myos create" -> set the command to "create".
 command=$1
-# cross-platform dir resolution
+
+# Set the path to myos
```
Comment (rylandg) [inline myos.sh]: Add .sh to “myos”

File: myos.sh [hunk 4]
```diff
@@ -1,92 +1,117 @@
...
 myosPath="$( cd "$(dirname  "$0")" ; pwd -P )"
-# follows symlink
+
+# Set the script name and its path
 scriptName="$(basename "$(test -L "$0" && readlink "$0" || echo "$0")")"
 scriptPath="$myosPath/$scriptName"
 
+# Declaring commands :
+# - Init : Creates the initial templates that the default docker-compose.yml expects.
+# - Build : Build the official MyOs image.
```
Comment (rylandg) [inline myos.sh]: Capitalize S in “MyOs”

**Comment by dimensi0n** [comment]
You're welcome

**Pull request merged**